internal/kldopenapi/abi2swagger.go

```diff
@@ -405,6 +418,11 @@ func (c *ABI2Swagger) addCommonParams(op *spec.Operation, isPOST bool, isConstru
 				Ref: privateForParam,
 			},
 		})
+		op.Parameters = append(op.Parameters, spec.Parameter{
```

Comment:
Could we add a parameter to `NewABI2Swagger` to include a boolean for `p.conf.OrionPrivateAPIS` so that we only add this parameter when running in Orion mode?

Reply:
done